assets/src/app.tsx

```diff
@@ -20,6 +21,11 @@ import ReactDOM from "react-dom"
 import ResizeObserver from "resize-observer-polyfill"
 import AppStateWrapper from "./components/appStateWrapper"
 
+Sentry.init({
+  dsn:
+    "https://e4ef550df0f644d5916bef033772db15@o89189.ingest.sentry.io/5303927",
```

Comment:
Does this mean that local development will send to sentry? If so, could we set this to only work in prod in the same way we configure fullstory and appcues?

Reply:
It does not, that's what the `included_environments` you noticed below will do.

Reply:
I'll look into it.